packages/e2e-tests/helpers/index.ts

```diff
@@ -14,6 +14,10 @@ export async function openDevToolsTab(page: Page) {
   }
 }
 
+export async function selectNodePicker(page: Page) {
+  await page.locator('[data-test-id="NodePicker"]').click();
+}
```

Comment:
nit: `helpers/index` is kind of just for top-level things.

This helper function should probably go in `helpers/elements-panel`. There's already a bit of code in there that works with that button that you could also replace 😄 
```js
const pickerButton = page.locator('[title="Select an element in the video to inspect it"]')!;
await pickerButton.click();
```

Reply:
Cleaned up! ✨